qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -775,3 +773,13 @@ void HOCRPdfExporter::updateValid() {
 	}
 	ui.buttonBox->button(QDialogButtonBox::Ok)->setEnabled(valid);
 }
+
+void HOCRPdfExporter::addInfoChanged() {
```

Comment:
addInfoClicked would be more appropriate I suppose

Reply:
Fixed